### PR TITLE
CMSP-323: CI Sync of default -> main

### PR DESCRIPTION
**Pull request by rwagner00**
Provides automatic syncing of the default branch to the main branch to facilitate automated testing and documentation to explain.

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.CI_ACCESS_TOKEN }}
+      - name: Sync default branch to main
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "<>"
```
Comment (jazzsequence) [inline .github/workflows/testing.yml:19]: We can use the pattern here that we've used [elsewhere](https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/devops/scripts/deploy-public-upstream.sh#L61-L62)

Comment (rwagner00) [reply .github/workflows/testing.yml]: Fixed

File: README.md [hunk 2]
```diff
@@ -136,6 +136,8 @@ Contributions are welcome in the form of GitHub pull requests. However, the `pan
 
 Instead, to propose a change, please fork [pantheon-systems/wordpress-composer-managed](https://github.com/pantheon-systems/wordpress-composer-managed) and submit a PR to that repository.
 
+Feature or bugfix development branches/pull requests should originate from and be submitted against the `default` branch. Please note that the `main` branch is the main _test_ branch and exists as a clone of the default branch for use in testing automation.
```
Comment (greg-1-anderson) [inline README.md:139]: I'm unclear why the `main` branch exists if it is always a copy of the default branch. Also, the comment above seems to be for the pantheon-systems/wordpress-composer-managed repo, whereas the comments above it appear to be for the pantheon-upstream/wordpress-composer-managed repo, and it's not clear to me which is out of place (or whether, perhaps, I am simply confused about the purpose of this change).

Comment (greg-1-anderson) [reply README.md:139]: It seems that you should have two README files, one that is displayed in this project, and one that is copied into pantheon-upstream/wordpress-composer-managed. This is already done for the decoupled upstream, but not for pantheon-upstream/wordpress-composer-managed.

Comment (rwagner00) [reply README.md]: Updated

File: devops/files/composer-managed-README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Composer-enabled Decoupled WordPress template
+
+This is an artifact generated by the [pantheon-systems/wordpress-composer-managed](https://github.com/pantheon-systems/wordpress-composer-managed) repo for the purposes of supporting Composer Managed WordPress. 
+
+## Contributing
+
+Contributions are welcome in the form of GitHub pull requests. However, the `pantheon-upstreams/decoupled-wordpress-composer-managed` repository is a mirror that does not directly accept pull requests.
```
Comment (greg-1-anderson) [inline devops/files/composer-managed-README.md:7]: Typo, remove "decoupled"

**Review by CdrMarks — DISMISSED** [review]

**Review by CdrMarks — APPROVED** [review]

**Pull request merged**